### ISSUE TITLE
Sensor for days since reset / date_recording_started

### ISSUE THREAD

**Issue by bgrr74** [hif2k1/battery_sim#138]
Is it possible to add a sensor for days since reset / date_recording_started
Or could even be an attribute under sensor.battery_sim where all the other attributes are located.

This will make it a lot easier to see how many days the battery_sim has been working since last reset / start

I've been trying to calculate the days in a template sensor but can't get i working :-(
My programming skills are not good enough :-(

**Comment by hif2k1** [comment]
Good idea. Sorry I've not had much time to work on this recently, but I
think this should be possible when I get the chance

**Comment by dewi-ny-je** [comment]
The battery entity already offers "Date recording started" among the attributes. Knowing the days since reset is an extremely uncommon use case, so a separate sensor makes little sense. You can create a custom sensor using the attribute.
